### ISSUE TITLE
02_packer  unable to use ami-f929abe8

### ISSUE THREAD

**Issue by clementcohen** [davmartini/ocp_outscale#2]
Hello, i'm facing to a issue while using the ami-f929abe8  :
![image](https://user-images.githubusercontent.com/2699448/189163822-c37b3dc4-1015-40a6-ba5d-28afecee465c.png)
The VM looks  up  & running but  the hostname is suposed to be ip-<reverse of public IPv4> isn't it? maybe  something wrong with the image? 
![image](https://user-images.githubusercontent.com/2699448/189165547-4e070ec9-215e-46f2-b733-9db506ac29e7.png)

**Comment by davmartini** [comment]
Hello
Witch user did you use for ssh connection? For Fedora 33 AMI, the user is fedora.

**Comment by clementcohen** [comment]
Hello David, 
thanks for the quick reply ,  I use fedora

here's my outscale.pkr.hcl file: 
`packer {
  required_plugins {
    outscale = {
      version = ">= 1.0.0"
      source = "github.com/hashicorp/outscale"
    }
  }
}

variable "access_key" {
  type =  string
  default = "XXXXX"
  // Sensitive vars are hidden from output as of Packer v1.6.5
  sensitive = true
}

variable "secret_key" {
  type =  string
  default = "XXXX"
  // Sensitive vars are hidden from output as of Packer v1.6.5
  sensitive = true
}

variable "region" {
  type =  string
  default = "eu-west-2"
}

variable "keypair_name" {
  type =  string
  default = "ocp-worker"
}

variable "the_sg" {
  type =  string
  default = "seg-ocp01-aza"
}

variable "omi_source" {
  type =  string
  default = "ami-f929abe8"
}

variable "ssh_path" {
  type =  string
  default = "/home/ccohen/.ssh/keys.d/outscale_ocp-worker.rsa"
}

variable "os_user" {
  type =  string
  default = "outscale"
}


source "outscale-bsusurrogate" "coreos" {
  region = var.region
  access_key = var.access_key
  secret_key = var.secret_key
  vm_type = "t2.medium"
  ssh_username = var.os_user
  communicator = "ssh"
  ssh_keypair_name = var.keypair_name
  ssh_private_key_file = var.ssh_path
  source_omi = var.omi_source
  omi_name = "omi_rhcos_user"
  launch_block_device_mappings {
    device_name = "/dev/xvdf"
    volume_size = 100
    volume_type = "io1"
    iops = 3000
    delete_on_vm_deletion = true
  }
  omi_root_device {
    source_device_name = "/dev/xvdf"
    device_name = "/dev/sda1"
    delete_on_vm_deletion = true
    volume_size = 100
    volume_type = "io1"
    iops = 3000
  }
}

build {
  name    = "coreos-outscale"
  sources = ["source.outscale-bsusurrogate.coreos"]
  provisioner "file" {
    source = "remote.ign"
    destination = "/tmp/remote.ign"
  }
  provisioner "shell" {
    inline = [
      "lsblk",
      "sudo yum install coreos-installer jq screen wget -y",
      "wget -c https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/latest/rhcos-metal.x86_64.raw.gz",
      "sudo coreos-installer install --insecure -f rhcos-metal.x86_64.raw.gz -i /tmp/remote.ign /dev/sda",
      "lsblk"
    ]
  }
}
`

**Comment by clementcohen** [comment]
woops I did   some testing and replace the user :(  but same error

**Comment by davmartini** [comment]
Two remarks, are you sure of your private SSH key path corresponding to your SSH Key used to deploy your temporary packer VM on outscale? Secondary, the security group used for this packer VM permit SSH access from your bastion or workstation?

**Comment by clementcohen** [comment]
> Two remarks, are you sure of your private SSH key path corresponding to your SSH Key used to deploy your temporary packer VM on outscale? 

Yes  

> Secondary, the security group used for this packer VM permit SSH access from your bastion or workstation?
![image](https://user-images.githubusercontent.com/2699448/189173201-77b1a4f6-4216-4478-8632-0ee3d9539623.png)

It's  automatically built by the provider  so, it's supposed to ...   maybe there's an issue  i'll try to reach outscale support

**Comment by clementcohen** [comment]
i'm able to loged in with  the key  manually,  journalctl logs show some errors: 
```sh
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[931]: CRYPTO_KEY_USER pid=931 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=destroy kind=server fp=SHA256:99:d2:e8:28:d9:29:45:7e:d7:c1:df:46:2a:9b:88:42:32:35:41:c5:5b:b1:45:50:04:68:f7:7d:e5:46:38:dc direction=? spid=931 suid=0  exe="/usr/sbin/sshd" hostname=? addr=? terminal=? res=success'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: CRYPTO_SESSION pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=start direction=from-server cipher=aes128-gcm@openssh.com ksize=128 mac=<implicit> pfs=curve25519-sha256@libssh.org spid=931 suid=74 rport=3702 laddr=10.8.7.118 lport=22  exe="/usr/sbin/sshd" hostname=? addr=82.64.250.128 terminal=? res=success'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: CRYPTO_SESSION pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=start direction=from-client cipher=aes128-gcm@openssh.com ksize=128 mac=<implicit> pfs=curve25519-sha256@libssh.org spid=931 suid=74 rport=3702 laddr=10.8.7.118 lport=22  exe="/usr/sbin/sshd" hostname=? addr=82.64.250.128 terminal=? res=success'
**Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal sshd[930]: userauth_pubkey: key type ssh-rsa not in PubkeyAcceptedKeyTypes [preauth]**
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: CRYPTO_KEY_USER pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=destroy kind=session fp=? direction=both spid=931 suid=74 rport=3702 laddr=10.8.7.118 lport=22  exe="/usr/sbin/sshd" hostname=? addr=82.64.250.128 terminal=? res=success'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal sshd[930]: Connection closed by authenticating user fedora 82.64.250.128 port 3702 [preauth]
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: CRYPTO_KEY_USER pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=destroy kind=server fp=SHA256:99:d2:e8:28:d9:29:45:7e:d7:c1:df:46:2a:9b:88:42:32:35:41:c5:5b:b1:45:50:04:68:f7:7d:e5:46:38:dc direction=? spid=931 suid=74  exe="/usr/sbin/sshd" hostname=? addr=? terminal=? res=success'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: USER_ERR pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=PAM:bad_ident grantors=? acct="?" exe="/usr/sbin/sshd" hostname=82.64.250.128 addr=82.64.250.128 terminal=ssh res=failed'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: CRYPTO_KEY_USER pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=destroy kind=server fp=SHA256:99:d2:e8:28:d9:29:45:7e:d7:c1:df:46:2a:9b:88:42:32:35:41:c5:5b:b1:45:50:04:68:f7:7d:e5:46:38:dc direction=? spid=930 suid=0  exe="/usr/sbin/sshd" hostname=? addr=? terminal=? res=success'
Sep 08 19:39:02 ip-10-8-7-118.eu-west-2.compute.internal audit[930]: USER_LOGIN pid=930 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=login acct="fedora" exe="/usr/sbin/sshd" hostname=? addr=82.64.250.128 terminal=ssh res=failed'
Sep 08 19:39:09 ip-10-8-7-118.eu-west-2.compute.internal sshd[932]: error: kex_exchange_identification: Connection closed by remote host
Sep 08 19:39:09 ip-10-8-7-118.eu-west-2.compute.internal audit[932]: CRYPTO_KEY_USER pid=932 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:sshd_t:s0-s0:c0.c1023 msg='op=destroy kind=server fp=SHA256:99:d2:e8:28:d9:29:45:7e:d7:c1:df:46:2a:9b:88:42:32:35:41:c5:5b:b1:45:50:04:68:f7:7d:e5:46:38:dc direction=? spid=932 suid=0  exe="/usr/sbin/sshd" hostname=? addr=? terminal=? res=success'
Sep 08 19:39:09 ip-10-8-7-118.eu-west-2.compute.internal sshd[932]: Connection closed by 82.64.250.128 port 3704
```

why key ssh-rsa is not in PubKey, it's  the same key I use manually...

**Comment by clementcohen** [comment]
found the issue :) @see https://github.com/outscale/packer-plugin-outscale/issues/77
